packages/vsce/__tests__/__unit__/utils/iconUtils.unit.test.ts

```diff
@@ -0,0 +1,108 @@
+import { getIconByStatus, getIconOpen, getIconPathInResources, getIconRootName } from "../../../src/utils/iconUtils";
+
+const iconPath = {
+  light: "program-dark.svg",
+  dark: "program-light.svg",
+};
+
+describe("Test suite for iconUtils", () => {
+  describe("Test suite for getIconPathResources", () => {
+    it("Should return icon path for dark and light theme", () => {
+      const result = getIconPathInResources("program");
+      expect(result.dark).toContain(iconPath.dark);
```

Comment:
Is `expect().toEqual()` more accurate than `.toContain()`?? We don't want it to contain anything other than that string right?

Reply:
I hit this difficulty - it's quite hard to form the exact string for the path because `__dirName__` in the test folders is different from `__dirName__` in the production code folder. So I ended up doing what Rushabh is - I did look at using a match method to say "must end with this string" but again I didn't quite get that working neatly.

Another option would be to refactor again so that only the part after `__dirName__` was generated but I don't like this so much.

I think a future step to verify that each generated name had a matching image would help make things watertight, r too.

Reply:
Okay cool thanks for clarifying that - makes sense. Does a `toEndWith` method exist? Although it's barely better than contains so not a huge deal.

Reply:
I'm probably nit-picking, let's all just be happy we're adding test coverage! 😎